Makefile

```diff
@@ -21,8 +21,8 @@ else ifeq ($(TEE_PLATFORM), fs)
   ATTESTER = none
 else ifeq ($(TEE_PLATFORM), tdx)
   ATTESTER = tdx-attester
-else ifeq ($(TEE_PLATFORM), az-tdx-vtpm)
-  ATTESTER = az-tdx-vtpm-attester
+else ifeq ($(TEE_PLATFORM), az-cvm-vtpm)
```

Comment:
while we are at it, perhaps change this one too https://github.com/confidential-containers/guest-components/blob/17f9899c226e5097d1157d86b37f8e5459bf39bc/.github/workflows/aa_cc_kbc.yml#L42

Reply:
actually, this seems different and a bit confusing. That is for `KBC` (which does not seem to be used in the Makefile :thinking:) looks like the github workflows did not get updated in c41e8c1022cff748abb0214cd23b83c2e342d11b

Reply:
yeah, I think the toggles are used consistently across the repo, some of them are historic artifacts. KBC, ATTESTERS and now TEE_PLATFORM are all variations of the same toggle with subtle differences. This probably should be cleaned up holistically in docs, makefiles and workflows at some point, but I'm hesitant to put more drive-by fixes into this PR.

I'll open another PR to add the azure tdx line that workflow.